addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/converter/swagger/RequestModelConverter.java

```diff
@@ -35,6 +38,7 @@
 
     private OperationModel operationModel;
     private Generators generators;
+    private static final Logger LOG = LogManager.getLogger(RequestModelConverter.class);
```

Comment:
This doesn't seem to be used in the new code?

Reply:
Right. I will remove it. Thanks.